host/xtest/CMakeLists.txt

```diff
@@ -18,13 +18,15 @@ endif()
 
 include(GNUInstallDirs)
 
+set (PYTHON3 python3)
+
 macro(EMBED_8100FILE prefix infile)
 	add_custom_command(
 		OUTPUT  regression_8100_${prefix}.h
-		COMMAND ${OPTEE_TEST_ROOT_DIR}/scripts/file_to_c.py --inf ${infile}
+		COMMAND ($(PYTHON3) ${OPTEE_TEST_ROOT_DIR}/scripts/file_to_c.py --inf ${infile})
```

Comment:
My cmake knowledge is also limited but have you tested this to make sure enclosing the `COMMAND` entry with `()` is ok?

Reply:
i enclosed within () because that command is one single entity

Reply:
Ok, but have you tested building with the `()` and made sure there are no errors?

Reply:
That doesn't look right indeed, the `--out` and `--name` arguments are clearly the continuation of the command.